src/operator/tensor/elemwise_binary_broadcast_op.h

```diff
@@ -33,23 +34,17 @@ inline bool BinaryBroadcastShape(const nnvm::NodeAttrs& attrs,
     SHAPE_ASSIGN_CHECK(*out_attrs, 0, lhs);
     return true;
   }
-
   TShape out(std::max(lhs.ndim(), rhs.ndim()));
-  std::fill_n(out.begin(), out.ndim(), 0);
   index_t bl = out.ndim() - lhs.ndim();
   index_t br = out.ndim() - rhs.ndim();
   for (index_t i = 0; i < out.ndim(); ++i) {
     index_t l = 1, r = 1;
     if (i >= bl) l = lhs[i-bl];
     if (i >= br) r = rhs[i-br];
     if (l != r) {
-      if (l == 0 || r == 0) {
```

Comment:
Ok. I think it's due to git history. Could you rebase on current master first?

git fetch upstream
git rebase upstream/master